docs/HISTORY.txt

```diff
@@ -18,6 +18,8 @@ Changelog
 - Modernize z3c form used in test by using directives to setup widgets and
   removing form wrapper. [buchi]
 
+- Add support for Python 3. [buchi]
```

Comment:
@buchi should we ship a `2.0.0` with `Plone 5.2` without Python 3 in order to find problems in the nightlies and then ship Python 3 later? It could simplify resolving problems when we can separate those two things.
What are your thoughts on this?

Reply:
I don't expect problems in other packages because of Plone 5.2 support as they do not have tests with Plone 5.2. Thus I would ship the 2.0.0 release with Python 3 support.

Reply:
ok 👍